types/priv_validator.go

```diff
@@ -90,12 +90,14 @@ func (pv MockPV) SignVote(ctx context.Context, chainID string, vote *tmproto.Vot
 	}
 
 	signBytes := VoteSignBytes(useChainID, vote)
+	extSignBytes := VoteExtensionSignBytes(useChainID, vote)
 	sig, err := pv.PrivKey.Sign(signBytes)
 	if err != nil {
 		return err
 	}
 	vote.Signature = sig
-	return nil
+	vote.ExtensionSignature, err = pv.PrivKey.Sign(extSignBytes)
```

Comment:
Consider adding an "if" just like in line 95. The difference is that, if you get an error, you don't assign `vote.ExtensionSignature`, as is done with `vote.Signature`.